deps/admin-worker-requirements.txt

```diff
@@ -31,7 +27,7 @@ billiard==4.2.1
     #   -c requirements.txt
     #   -c worker-requirements.txt
     #   celery
-celery[redis]==5.4.0
+celery==5.4.0
```

Comment:
See https://github.com/astral-sh/uv/issues/1595 for rational and some background. TLDR: It shouldn't make a difference.

There's some more chatter here too(Is also linked in the uv issue above) - https://github.com/astral-sh/uv/issues/1595